### ISSUE TITLE
09 Publisher

### ISSUE THREAD

**Issue by baskaufs** [HeardLibrary/twine-for-wikicite#40]
Focused on adding a publisher value to an existing work.

**Comment by baskaufs** [comment]
@CliffordAnderson @cylew @awesolek2 @Talinum Just FYI I have had a setback today in that my work computer basically died and is in the process of getting resurrected by VUIT. I'm mostly functional on my personal computer but will have to download Twine again, etc. plus lost about a half days worth of work today thrashing with the bad boy. So I may be a bit slow in the processing.

**Comment by baskaufs** [comment]
From Charlotte 2021-03-23:
To be able to create a new item for a publisher, please try “West Academic Publishing.”
Find details in Books in Print and publisher website. If you want to list publisher ISBN prefix as an identifier, Books in Print is a great source. Let me know if you have questions.

Website URL: https://home.westacademic.com/legal-publisher

**Comment by baskaufs** [comment]
@CliffordAnderson @cylew @awesolek2 @Talinum The publisher script draft is up now in Box. I'm not that happy with it, but I suppose it's OK. To a large extent, it repeats information covered in other videos, but it does serve the purpose of suggesting reasons for going off on three different learning tracks that we talked about (gadgets, using paid reference services, and using open resources).

**Comment by baskaufs** [comment]
@cylew @awesolek2 @Talinum  The publisher script is finalized.

**Comment by awesolek2** [comment]
@CliffordAnderson Chinese Publisher animated

**Comment by awesolek2** [comment]
@CliffordAnderson English mp4, wav, and animation added to box

**Comment by awesolek2** [comment]
@CliffordAnderson Spanish publisher animated

**Comment by CliffordAnderson** [comment]
@awesolek2 The Chinese Character Animator project seems to be broken. I cannot import the project from the files on Box.

**Comment by awesolek2** [comment]
@CliffordAnderson I just uploaded a compressed copy, maybe that will work? It's opening fine on my end. These projects are super finicky!

**Comment by CliffordAnderson** [comment]
@awesolek2 Thanks. Let me try again.

**Comment by CliffordAnderson** [comment]
@awesolek2 It worked that time! Video coming soon....

**Comment by CliffordAnderson** [comment]
@baskaufs Videos for Chinese, English, and Spanish are now available on Box

**Comment by baskaufs** [comment]
The publisher video 09 has been uploaded and embedded in the Twine passage. However, it is missing the Chinese and Spanish TLDR text and the Chinese video is lacking a title on YouTube so it has an English title when you see it in the iFrame. I'm not going to close this issue yet since the passage still needs these things to be complete. @Talinum and @cylew, when you've put those things in the learning-objectives.docx document in Box, please ping me and I'll put them in the passage and push a new copy of the website to GitHub.

**Comment by CliffordAnderson** [comment]
@cylew @ New version of Chiense video on Box.

**Comment by baskaufs** [comment]
Chinese video updated in YouTube and Twine with newest version, but YouTube metadata still has only an English title.

**Comment by baskaufs** [comment]
videos added to twine, cover metadata problems in another issue
